test/base-fork/Morpho.t.sol

```diff
@@ -140,50 +140,45 @@ contract MorphoDepositFailureTests is MorphoBaseTest {
         foreignController.depositERC4626(makeAddr("fake-token"), 1e18);
     }
 
-    function test_depositERC4626_zeroMaxSlippage() external {
-        vm.prank(Base.SPARK_EXECUTOR);
-        foreignController.setMaxSlippage(MORPHO_VAULT_USDS, 0);
+    function test_depositERC4626_exchangeRateBoundary() external {
+        deal(Base.USDS, address(almProxy), 25_000_000e18);
+
+        vm.startPrank(Base.SPARK_EXECUTOR);
+        foreignController.setMaxExchangeRate(MORPHO_VAULT_USDS, 1e18, IERC4626(MORPHO_VAULT_USDS).convertToAssets(1e18) - 1);
+        vm.stopPrank();
+
+        vm.prank(relayer);
+        vm.expectRevert("ForeignController/exchange-rate-too-high");
+        foreignController.depositERC4626(MORPHO_VAULT_USDS, 25_000_000e18);
+
+        vm.startPrank(Base.SPARK_EXECUTOR);
+        foreignController.setMaxExchangeRate(MORPHO_VAULT_USDS, 1e18, IERC4626(MORPHO_VAULT_USDS).convertToAssets(1e18));
+        vm.stopPrank();
 
         vm.prank(relayer);
-        vm.expectRevert("ForeignController/max-slippage-not-set");
-        foreignController.depositERC4626(MORPHO_VAULT_USDS, 1e18);
+        foreignController.depositERC4626(MORPHO_VAULT_USDS, 25_000_000e18);
     }
 
     function test_morpho_usds_deposit_rateLimitedBoundary() external {
         deal(Base.USDS, address(almProxy), 25_000_000e18 + 1);
 
         vm.expectRevert("RateLimits/rate-limit-exceeded");
-        vm.startPrank(relayer);
+        vm.prank(relayer);
```

Comment:
Revert back to startPrank

Reply:
also prank should be before `vm.expectRevert`